docs/pushpayloads.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "title":"FxA Push Payload schema",
+  "description":"This schema defines what is acceptable to send as a payload data with the Push API from the FxA servers to a device connected to FxA",
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "type":"object",
+  "allOf":[
+    {
+      "type":"object",
+      "required":[
+        "version",
+        "command"
+      ],
+      "properties":{
+        "version":{
+          "type":"integer",
+          "description":"The version of this push payload data instance. Bump this if you make changes to any part of this schema."
+        },
+        "command":{
+          "type":"string",
+          "description":"Helps the receiving device discriminate payloads"
+        }
+      }
+    },
+    {
+      "type":"object",
+      "anyOf":[
+        { "$ref":"#/definitions/newDevice" },
+        { "$ref":"#/definitions/collectionChanged" }
+      ]
+    }
+  ],
+  "definitions":{
+    "newDevice":{
+      "type":"object",
+      "required":[
+        "data"
+      ],
+      "properties":{
+        "command":{
+          "enum":[
+            "fxaccounts:new_device"
+          ]
+        },
+        "data":{
+          "type":"object",
+          "required":[
+            "deviceName"
+          ],
+          "properties":{
+            "deviceName":{
+              "type":"string",
+              "description":"The name of the device who joined this account"
+            }
+          }
+        }
+      }
+    },
+    "collectionChanged":{
```

Comment:
Nit: if the body is a list of collections, this feels like it should be named plural "collections" rather than "collection"